panopticum/models.py

```diff
@@ -340,96 +382,6 @@ class ComponentVersionModel(models.Model):
 
     dev_api_is_public = panopticum.fields.NoPartialYesField("API is public")
 
-    # TODO: deprecated. Let's remove
-    # compliance
-
-    compliance_applicable = models.BooleanField(verbose_name="Compliance requirements are applicable", default=True)
```

Comment:
Do we still support this feature? When owner just marks the whole set of requirements as "n/a" (not applicable)

Reply:
We will. But not in current implementation. For example we can implement field many-to-many ```requirementsets```

Reply:
Ok, let's fix it later